### PR TITLE
Fix typo in example code

### PR DESCRIPTION
**Pull request by biggianteye**

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -676,4 +676,4 @@ A few additional options have been added:
 **headerContentSections** / **footerContentSections** - optional help sections that will appear before / after the `Options` section that is generated from the option config. In most cases you should probably include one header section that explains what the application does.  
 
 [//]: ####ts-command-line-args_generated-by-footer
-Markdown Generated by [ts-command-line-args](https://www.npmjs.com/package/ts-command-line-args)
\ No newline at end of file
+Markdown Generated by [ts-command-line-args](https://www.npmjs.com/package/ts-command-line-args)
```
Comment (Roaders) [inline README.MD:679]: can you remove this appended line please

**Review by Roaders — CHANGES_REQUESTED** [review]

File: README.MD (hunk 1 above)
Comment (biggianteye) [reply README.MD:679]: It's a newline character at the end of the file that the web editor automatically added. Do you still want it removed?

Comment (Roaders) [reply README.MD:679]: no, sorry. that's fine. I obviously mis-read the diff!

Comment (biggianteye) [reply README.MD]: It's gone now anyway. The build pipeline failed and wanted me to run "npm run write-markdown" which has removed the trailing newline.

**Comment by Roaders** [comment]
thanks for this. Sorry it took me a while to merge!

**Pull request merged**